src/cms/views/pages/page_actions.py

```diff
@@ -163,9 +163,9 @@ def link_callback(uri, rel):
 # pylint: disable=unused-argument, too-many-locals
 def export_pdf(request, region_slug, language_code):
     """
-    Function for handling a pdf export request for a single page.
-    If the page is a root page, the corresponding django template
-    inserts the content of all children pages.
+    Function for handling a pdf export request for pages.
+    The pages were either selected by cms user (see :func:`cms.static.js.pages.page_bulk_action.bulk_action_execute`)
```

Comment:
You can only reference Python classes/functions/objects with Sphinx, no JavaScript functions.

Reply:
Yes, until now this won't work. If you like, I could add sphinx-js to our pipfile, this should enable auto-generating docs from the js files too? See [https://hacks.mozilla.org/2017/07/introducing-sphinx-js-a-better-way-to-document-large-javascript-projects/](url)

Reply:
Very cool, I didn't know of this extension. I will open a new issue for this feature.